src/taco-remote-lib/resources/en/resources.json

```diff
@@ -68,6 +68,8 @@
     "_UnsupportedPlatform.comment": "Message reported to the client when a request for an unsupported platform is made",
     "NoCordovaVersionSpecified": "No Cordova version specified!",
     "_NoCordovaVersionSpecified.comment": "Error reported when trying to build without any version specified",
+    "UnsupportedCordovaAndNodeVersion": "Unable to build using Cordova earlier than 5.3.3 when on node version 4.0.0 and higher and also specifying deployment-target or target-device preferences in config.xml",
```

Comment:
strings need to be reviewed.
May be "Unable to build incompatible configuration: 4.0<= cordova < 5.3, Node < 4.0, deployment-target: true, target-device"